### PR TITLE
Add answer update

### PR DESCRIPTION
**Pull request by kssampson**
- Fixed missing Reported functionality lost in previous push
- Have put & get for Add Answer component

### CODE REVIEW

**Comment by kssampson** [comment]
@gordon1213 @LukeLarson2 @clee4037

File: client/src/components/questions_answers/AddAnswer.jsx [hunk 1]
```diff
@@ -2,22 +2,19 @@ import React, { useEffect, useState } from 'react';
...
+  // }
 
-  useEffect(() => {
-    axios.get(`products/?product_id=${itemId}`)
-    .then((response) => {
-      setProduct(response.data)
-    })
-    .catch((err) => console.error(err))
-  }, [itemId])
+  console.log('questionId in AddAnswer: ', questionId)
```
Comment (clee4037) [inline client/src/components/questions_answers/AddAnswer.jsx]: Delete or comment out please!

**Review by clee4037 — COMMENTED** [review]
Looks good!

File: client/src/components/questions_answers/AddAnswer.jsx [hunk 2]
```diff
@@ -2,22 +2,31 @@ import React, { useEffect, useState } from 'react';
...
+    axios.post(`/qa/questions/${questionId}/answers`, {
+      body: yourAnswer,
+      name: yourNickName,
+      email: yourEmail,
+      photos: yourPhotos
     })
-    .catch((err) => console.error(err))
-  }, [itemId])
+    .then((response) => {
+      console.log('response inside AddAnswer PUT', response)
```
Comment (LukeLarson2) [inline client/src/components/questions_answers/AddAnswer.jsx:23]: Please remove console log

File: client/src/components/questions_answers/Answers.jsx [hunk 3]
```diff
@@ -6,18 +6,35 @@ import '../../stylesheets/questions_answers/answers.css'
 const Answers = ( { questionId, answerData, setAnswerData } ) => {
 
   const [marked, setMarked] = useState({});
+  const [reported, setReported] = useState({});
 
 
   const checkMarked = (id) => {
+    console.log('inside check marked')
```
Comment (LukeLarson2) [inline client/src/components/questions_answers/Answers.jsx:13]: Please remove console log

**Review by LukeLarson2 — CHANGES_REQUESTED** [review]
Please remove console logs mentioned in comments before approval

**Comment by kssampson** [comment]
removed logs

**Review by LukeLarson2 — APPROVED** [review]
Thank you for removing console logs! Good to go

**Pull request merged**